src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- Don't run .NET Framework only tests on non Windows. -->
+    <SkipTests Condition="'$(OS)' != 'Windows_NT'">true</SkipTests>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(CommonPath)\System\MutableDecimal.cs" Link="Common\System\MutableDecimal.cs" />
+    <Compile Include="$(CommonPath)tests\System\Buffers\NativeMemoryManager.cs" Link="Common\System\Buffers\NativeMemoryManager.cs" />
+    <Compile Include="$(CommonPath)tests\System\Diagnostics\Tracing\TestEventListener.cs" Link="Common\System\Diagnostics\Tracing\TestEventListener.cs" />
+    <Compile Include="$(CommonPath)tests\TestUtilities\System\AssertExtensions.cs" Link="Common\System\AssertExtensions.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) != '.NETCoreApp'">
+    <!-- Tests specific to the fast span -->
+    <Compile Remove="MemoryMarshal\CreateSpan.cs" />
+    <Compile Remove="MemoryMarshal\CreateReadOnlySpan.cs" />
+    <Compile Remove="Span\Reflection.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.Memory.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.RemoteExecutor" />
+    <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
```

Comment:
XUnitExtensions will be used by all test projects so it would be good to also move that out into the root Directory.Build file and condition it on IsTestProject.

Reply:
I see RemoveExecutor also imported in the 3 test projects. I'll move it as well.